### PR TITLE
doc/rados: edit "Everything Failed! Now What?"

### PR DESCRIPTION
**Pull request by zdover23**
Edit the section "Everything Failed! Now what?" in doc/rados/troubleshooting/troubleshooting-mon.rst.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/rados/troubleshooting/troubleshooting-mon.rst [hunk 1]
```diff
@@ -628,16 +627,20 @@ irc.oftc.net), or at ``dev@ceph.io`` and ``ceph-users@lists.ceph.com``. Make
 sure that you have prepared your logs and that you have them ready upon
 request.
 
-See https://ceph.io/en/community/connect/ for current (as of October 2023)
-information on getting in contact with the upstream Ceph community.
+The Upstream Ceph Slack can be joined at this address:
```
Comment (anthonyeleven) [inline doc/rados/troubleshooting/troubleshooting-mon.rst:630]: The Ceph Slack workspace may be joined

File: doc/rados/troubleshooting/troubleshooting-mon.rst [hunk 2]
```diff
@@ -628,16 +627,20 @@ irc.oftc.net), or at ``dev@ceph.io`` and ``ceph-users@lists.ceph.com``. Make
...
+https://ceph-storage.slack.com/
 
+See https://ceph.io/en/community/connect/ for current (as of December 2023)
+information on getting in contact with the upstream Ceph community.
```
Comment (anthonyeleven) [inline doc/rados/troubleshooting/troubleshooting-mon.rst:634]: S/upstream//

File: doc/rados/troubleshooting/troubleshooting-mon.rst [hunk 3]
```diff
@@ -648,32 +651,34 @@ cluster's configuration files. If Ceph is using the default debug levels, then
...
+other components, the Monitors have different parts that output their debug
 information on different subsystems.
 
 If you are an experienced Ceph troubleshooter, we recommend raising the debug
-levels of the most relevant subsystems. Of course, this approach might not be
-easy for beginners. In most cases, however, enough information to address the
-issue will be secured if the following debug levels are entered::
+levels of the most relevant subsystems. This approach might not be easy for
+beginners. In most cases, however, enough information to address the issue will
+be secured if the following debug levels are entered::
```
Comment (anthonyeleven) [inline doc/rados/troubleshooting/troubleshooting-mon.rst:662]: S/secured/logged/

File: doc/rados/troubleshooting/troubleshooting-mon.rst [hunk 4]
```diff
@@ -648,32 +651,34 @@ cluster's configuration files. If Ceph is using the default debug levels, then
...
 
       debug_mon = 10
       debug_ms = 1
 
 Sometimes these debug levels do not yield enough information. In such cases,
-members of the upstream Ceph community might ask you to make additional changes
+members of the upstream Ceph community will ask you to make additional changes
```
Comment (anthonyeleven) [inline doc/rados/troubleshooting/troubleshooting-mon.rst:668]: S/upstream//

File: doc/rados/troubleshooting/troubleshooting-mon.rst [hunk 5]
```diff
@@ -712,17 +719,17 @@ or:
...
-which portions are relevant so we are willing to accept complete and
-unabridged logs. However, we request that you avoid sending logs containing
-hundreds of thousands of lines with no additional clarifying information. One
-common-sense way of making our task easier is to write down the current time
-and date when you are reproducing the problem and then extract portions of your
-logs based on that information.
-
-Finally, reach out to us on the mailing lists or IRC or Slack, or by filing a
-new issue on the `tracker`_.
+Send the upstream Ceph community only the portions of your logs that are
```
Comment (anthonyeleven) [inline doc/rados/troubleshooting/troubleshooting-mon.rst:722]: S/upstream//

Just one community, so we don’t need to qualify?

File: doc/rados/troubleshooting/troubleshooting-mon.rst [hunk 6]
```diff
@@ -712,17 +719,17 @@ or:
...
+relevant to your Monitor problems. Because it might not be easy for you to
+determine which portions are relevant, the upstream Ceph community accepts
+complete and unabridged logs. But don't send logs containing hundreds of
+thousands of lines with no additional clarifying information. One common-sense
+way of making life easier for the upstream Ceph community is to write down the
```
Comment (anthonyeleven) [inline doc/rados/troubleshooting/troubleshooting-mon.rst:727]: Way to help the Ceph community help you is to

File: doc/rados/troubleshooting/troubleshooting-mon.rst (hunk 5 above)
Comment (zdover23) [reply doc/rados/troubleshooting/troubleshooting-mon.rst:722]: I want to distinguish the upstream community  from communities for which a person might pay for support.

File: doc/rados/troubleshooting/troubleshooting-mon.rst (hunk 6 above)
Comment (zdover23) [reply doc/rados/troubleshooting/troubleshooting-mon.rst:727]: Accepted.

File: doc/rados/troubleshooting/troubleshooting-mon.rst (hunk 4 above)
Comment (zdover23) [reply doc/rados/troubleshooting/troubleshooting-mon.rst:668]: See the comment below. TL;DR: I want "upstream" to distinguish the Foundation-led community from the companies that might have their own, pay-for-admission communities.

File: doc/rados/troubleshooting/troubleshooting-mon.rst (hunk 3 above)
Comment (zdover23) [reply doc/rados/troubleshooting/troubleshooting-mon.rst:662]: Accepted.

File: doc/rados/troubleshooting/troubleshooting-mon.rst (hunk 1 above)
Comment (zdover23) [reply doc/rados/troubleshooting/troubleshooting-mon.rst:630]: Accepted with slight modification.

File: doc/rados/troubleshooting/troubleshooting-mon.rst (hunk 5 above)
Comment (anthonyeleven) [reply doc/rados/troubleshooting/troubleshooting-mon.rst:722]: I wouldn't call the latter a _community_ but you're the boss, head man, top dog, big cheese, the head honcho, number...

Comment (zdover23) [reply doc/rados/troubleshooting/troubleshooting-mon.rst:722]: I chose the wrong week to stop sniffing glue.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/54665 - Reef backport
https://github.com/ceph/ceph/pull/54666 - Quincy backport

**Pull request merged**